### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by onyxraven**
Background
-----

* `go get -u && go tidy` to update dependencies
  * cobra and the md generator for cobra were updated
* github actions updates
* lint fixes
* version agreement in go.mod

This is in prep to work on the pending PR #61 

Versioning
-----

This PR is at most a patch, but probably will roll up into the next update (no criticals)

Tasks
-----

* [ ] Manual testing

### CODE REVIEW

**Comment by ibcheckmarx** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [664d8f65\-d83d\-4f46\-8948\-be5e305d5ce2](https://ast.checkmarx.net/projects/38015e52-128b-4c5b-abb5-db597bcbc6bd/scans?id=664d8f65-d83d-4f46-8948-be5e305d5ce2&branch=dependency-updates)
### No New Or Fixed Issues Found

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: "1.20"
```
Comment (ibottamike) [inline .github/workflows/lint.yaml:16]: The go-version was not changed as part of this PR, but curious if we should have the version in go.mod (`go 1.19`) and version referenced in gitHub actions match.  Low priority as this is lint, more curious than anything.

**Review by ibottamike — DISMISSED** [review]
Looks good, assuming go mod tidy did its thing for the version.  One small question on version mismatch.

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (onyxraven) [reply .github/workflows/lint.yaml:16]: ah, ya, i can get those synced up

**Pull request merged**